data/json/items/armor/head_attachments.json

```diff
@@ -61,9 +61,9 @@
     "material": [ "thermo_resin" ],
     "symbol": "[",
     "color": "dark_gray",
-    "material_thickness": 1,
+    "material_thickness": 1.5,
     "flags": [ "HELMET_MANDIBLE_GUARD", "CANT_WEAR" ],
-    "armor": [ { "encumbrance": 12, "coverage": 100, "covers": [ "mouth" ] } ]
+    "armor": [ { "encumbrance": 10, "coverage": 100, "covers": [ "mouth" ] } ]
```

Comment:
Not sure it needs to be less encumbering *and* more protective. This feels out of place for this PR, if you want to rebalance this gear you should probably do it in a different PR dedicated to this. Same for the two other similar buffs